src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -202,64 +202,9 @@ private static IOperation GetDelegateCreationInstance(BoundDelegateCreationExpre
             return null;
         }
 
-        private static readonly ConditionalWeakTable<BoundObjectCreationExpression, object> s_memberInitializersMappings =
-            new ConditionalWeakTable<BoundObjectCreationExpression, object>();
-
-        private static ImmutableArray<ISymbolInitializer> GetObjectCreationMemberInitializers(BoundObjectCreationExpression expression)
+        private static ImmutableArray<IOperation> GetObjectCreationMemberInitializers(BoundObjectCreationExpression expression)
```

Comment:
I still see the original name in the C# factory `GetObjectCreationMemberInitializers`? You can probably fix it up in a follow up PR..